test/acceptance/setup-terraform/ecr.tf

```diff
@@ -8,7 +8,8 @@ locals {
 }
 
 resource "aws_ecr_repository" "lambda-registrator" {
-  name = local.ecr_repository_name
+  name         = local.ecr_repository_name
```

Comment:
I think that unconditionally creating the ecr repo in the test setup means that we never actually test the case where our module creates the repo :thinking:

Reply:
this will not get reused.this is for inital setup for manual approach. 
while testing autopublish new ecr_repo with name `consul-lambda-registrator-${random_id.repo_id.hex}` will get created